torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -69,3 +70,340 @@ void flag_to_float(const T* src, float* dst, int64_t n) {
     dst_u32[i] = *(src + i) ? 0xFFFFFFFF : 0;
   }
 }
+
+#if defined(CPU_CAPABILITY_AVX512)
+// TODO(jgong5): rewrite with ATEN vectorized (need to add unpack and shuffle)
+// Code from FBGEMM:
+// https://github.com/pytorch/FBGEMM/blob/39a423e4ad1a04b77fea81c7d09c3e6f8984fae9/src/UtilsAvx512.cc#LL19C6-L19C6
+// 16 * 6 = 96 instructions
+inline void transpose_kernel_16x16_avx512(
```

Comment:
It feels like this should be part of `at::vec` as opposed to in this header.

Reply:
Moved to `at::vec`.